mscore/selectnotedialog.cpp

```diff
@@ -49,7 +49,8 @@ SelectNoteDialog::SelectNoteDialog(const Note* _n, QWidget* parent)
       pitch->setText(n->tpcUserName());
       string->setText(QString::number(n->string()+1));
       type->setText(n->noteTypeUserName());
-      duration->setText(n->chord()->durationUserName());
+      durationType->setText(tr("%1 Note").arg(n->chord()->durationType().durationTypeUserName()));
```

Comment:
Not really sure wherther here it should be "Note" or "note".

Reply:
Or whether that text is needed at all. "Same note type: 16th" (rather than "Same note type: 16th note") and "Same note duration" Triplet 16th" should be enough

Reply:
Or even just "Same type: 16th" and "Same duration: Triplet 16th"

Reply:
When "Note" is omitted, normal notes (not tuplets) would have the same text for "Same note type" and "Same duration" (second picture at the top of this page, removing the "Note").
Without the "Note", it might be hard to understand the two different results for the two checkboxes.